packages/katello/rubygem-katello/rubygem-katello.spec

```
@@ -3,13 +3,13 @@
 %{!?scl:%global pkg_name %{name}}
 
 %global foreman_min_version 2.4
```

Should this be 2.5 to match?
```suggestion
%global foreman_min_version 2.5
```